subprojects/launcher/src/main/java/org/gradle/launcher/daemon/client/DaemonClient.java

```diff
@@ -159,15 +170,15 @@ protected Object executeBuild(Build build, DaemonClientConnection connection, Bu
             throw new DaemonInitialConnectException("The first result from the daemon was empty. Most likely the process died immediately after connection.");
         }
 
-        LOGGER.info("Received result {} from daemon {} (build should be starting).", result, connection.getDaemon());
+        LOGGER.debug("Received result {} from daemon {} (build should be starting).", result, connection.getDaemon());
 
         DaemonDiagnostics diagnostics = null;
         if (result instanceof BuildStarted) {
             diagnostics = ((BuildStarted) result).getDiagnostics();
             result = monitorBuild(build, diagnostics, connection, cancellationToken, buildEventConsumer);
         }
 
-        LOGGER.info("Received result {} from daemon {} (build should be done).", result, connection.getDaemon());
+        LOGGER.debug("Received result {} from daemon {} (build should be done).", result, connection.getDaemon());
```

Comment:
I think these are all OK to downgrade. They were added when we had a very strange concurrent bug when starting a daemon.